src/plugins/data/common/index_patterns/types.ts

```diff
@@ -149,7 +149,7 @@ export interface FieldSpecExportFmt {
 }
 
 export interface FieldSpec {
-  count: number;
+  count?: number;
```

Comment:
Looking at the changes in `src/plugins/data/common/index_patterns/fields/index_pattern_field.ts`, it seems that both functions can't return `undefined`. Can it be that you don't have to make these optional?

Reply:
@lizozom I'm making them optional so that someone defining a field doesn't have the burden of providing them - the expected defaults are obvious. Its not a necessary change but I think its still helpful.